src/configuration/testcafe-configuration.ts

```diff
@@ -34,7 +34,12 @@ import WarningLog from '../notifications/warning-log';
 import browserProviderPool from '../browser/provider/pool';
 import BrowserConnection, { BrowserInfo } from '../browser/connection';
 
-const CONFIGURATION_FILENAME = '.testcaferc.json';
+const BASE_CONFIGURATION_FILENAME       = '.testcaferc';
+const PRIORITY_CONFIGURATION_EXTENSIONS = {
```

Comment:
It can be declared just as array. Index of array is a priority of configuration.